src/extensions/default/HTMLCodeHints/main.js

```diff
@@ -319,10 +318,9 @@ define(function (require, exports, module) {
                 
                 // If we're at an attribute value, check if it's an attribute name that has hintable values.
                 if (this.tagInfo.attr.name) {
-                    var hintsAndSortFunc = this._getValueHintsForAttr({queryStr: query},
-                                                                      this.tagInfo.tagName,
-                                                                      this.tagInfo.attr.name);
-                    var hints = hintsAndSortFunc.hints;
+                    var hints = this._getValueHintsForAttr({queryStr: query},
```

Comment:
@humphd It would be good to reformat this statement so the diff shows only the change in the variable name.